frontend/src/components/Table/Table.tsx

```diff
@@ -353,7 +355,24 @@ const Table = <T extends IIdentifiedObject, TFilter extends object = {}>(
                     <IndeterminateCheckbox
                       {...row.getToggleRowSelectedProps()}
                       row={row}
-                      setSelected={setExternalSelectedRows}
+                      setSelected={(values: T[], blah: any) => {
```

Comment:
blah?

Reply:
oops

Reply:
I still see blah :(

Reply:
sorry, think I forgot to push. Doing that now